lib/galaxy/jobs/runners/chronos.py

```diff
@@ -115,8 +115,13 @@ def __init__(self, app, nworkers, **kwargs):
             kwargs[self.RUNNER_PARAM_SPEC_KEY] = {}
         kwargs[self.RUNNER_PARAM_SPEC_KEY].update(self.RUNNER_PARAM_SPEC)
         super(ChronosJobRunner, self).__init__(app, nworkers, **kwargs)
+        protocol = 'http' if self.runner_params.get('insecure', True)\
+            else 'https'
         self._chronos_client = chronos.connect(
-            self.runner_params['chronos'])
+            self.runner_params['chronos'],
+            username=self.runner_params.get('username'),
```

Comment:
`get(key)` returns `None` by default, if the key is not found, so there is no problem. :)